Makefile.include

```diff
@@ -319,6 +327,7 @@ endif
 ifneq ($(MAKECMDGOALS),clean)
 -include ${addprefix $(OBJECTDIR)/,$(CONTIKI_SOURCEFILES:.c=.d) \
                                    $(PROJECT_SOURCEFILES:.c=.d)}
+-include ${addprefix $(OBJECTDIR)/,$(PROJECT_SOURCEFILES:.cpp=.d)}
```

Comment:
So we expect Contiki-NG to stay C only, and we only allow project source files to be CPP.

No problem with me at all, for now at least. Perhaps in the future we can revisit if required!